### PR TITLE
fix exchange sender use DBMS_TCP_PROTOCOL_VERSION

### PR DESCRIPTION
**Pull request by Aaaaaaron**
I hereby agree to the terms of the CLA available at: [placeholder]

Changelog category (leave one):

- Bug Fix
- Not for changelog (changelog entry is not required)

Detailed description / Documentation draft:
wrong usage of ClickHouseRevision::getVersionRevision
...

If you are doing this for the first time, it's recommended to read the lightweight [Contributing to ByConity Documentation](https://github.com/ByConity/ByConity/blob/master/CONTRIBUTING.md) guide first.

### CODE REVIEW

**Comment by WillemJiang** [comment]
@Aaaaaaron  Please signed the CLA first.

**Comment by Aaaaaaron** [comment]
> @Aaaaaaron Please signed the CLA first.

done

File: src/Processors/Exchange/DataTrans/Brpc/BrpcRemoteBroadcastSender.cpp [hunk 1]
```diff
@@ -159,14 +159,14 @@ void BrpcRemoteBroadcastSender::serializeChunkToIoBuffer(Chunk chunk, WriteBuffe
         const CompressionCodecPtr & codec = CompressionCodecFactory::instance().get(method, level);
         CompressedWriteBuffer compressed_out(out, codec, DBMS_DEFAULT_BUFFER_SIZE * 2);
         NativeChunkOutputStream chunk_out(
-            compressed_out, ClickHouseRevision::getVersionRevision(), header, !settings.low_cardinality_allow_in_native_format);
+            compressed_out, DBMS_TCP_PROTOCOL_VERSION, header, !settings.low_cardinality_allow_in_native_format);
```
Comment (canhld94) [inline src/Processors/Exchange/DataTrans/Brpc/BrpcRemoteBroadcastSender.cpp:162]: Can you elaborate the changes here? Before we has this change does `BrpcRemoteBroadcastSender` stills work as expected?

Comment (hustnn) [reply src/Processors/Exchange/DataTrans/Brpc/BrpcRemoteBroadcastSender.cpp:162]: Hi @Aaaaaaron Any update on the reason of the fix?

Comment (Aaaaaaron) [reply src/Processors/Exchange/DataTrans/Brpc/BrpcRemoteBroadcastSender.cpp:162]: just wrong usage of ClickHouseRevision::getVersionRevision, it's common use DBMS_TCP_PROTOCOL_VERSION 
 not getVersionRevision

**Pull request closed without merging**